packages/kit/src/exports/node/polyfills.js

```diff
@@ -21,16 +26,21 @@ const globals = {
 };
 
 // exported for dev/preview and node environments
-// TODO: remove this once we only support Node 18.11+ (the version multipart/form-data was added)
 /**
  * Make various web APIs available as globals:
  * - `crypto`
- * - `fetch`
- * - `Headers`
- * - `Request`
- * - `Response`
+ * - `fetch` (only in node 18.11-)
+ * - `Headers` (only in node 18.11-)
+ * - `Request` (only in node 18.11-)
+ * - `Response` (only in node 18.11-)
  */
 export function installPolyfills() {
+	const version = process.versions.node.split('.').map((n) => parseInt(n, 10));
+	const globals =
```

Comment:
To preserve current behavior I gone for not bailing and instead always patching in case the check goes wrong

Reply:
I'd probably be in favor of a try-catch block here instead of all of these `?.`s, I think. There's a lot more that can go wrong than something being undefined.

Reply:
done